docs/commands/push.md

```diff
@@ -276,6 +277,28 @@ For more information on how to configure access to your APIs, check the [registr
 redocly push openapi/petstore.yaml @openapi-org/petstore-api@v1 --public
 ```
 
+### Files
+
+The `--files` option allows you to upload other folders and files.
+
+```bash
+redocly push openapi/petstore.yaml @openapi-org/petstore-api@v1 --files ./path/to/folder
+```
+
+You can also add files and folders providing them in `.redocly.yaml`:
+
+```yaml
+apis:
+  main: 
+     root: ./openapi.yam;
```

Comment:
do we support files in `apis` section?

Reply:
I think we should. Files might be specific to a specific API?

Reply:
Should we show an example of this in the docs?